packages/components/src/components/post-tabs/post-tabs.tsx

```diff
@@ -21,9 +21,11 @@ export class PostTabs {
   private hiding: Animation;
   private isLoaded = false;
 
-  private get tabs(): NodeListOf<HTMLPostTabHeaderElement> {
-    return this.host.querySelectorAll('post-tab-header');
-  }
+  private get tabs(): HTMLPostTabHeaderElement[] {
+    return Array.from(this.host.querySelectorAll<HTMLPostTabHeaderElement>('post-tab-header')).filter(tab =>
+      tab.closest('post-tabs') === this.host
+    );
```

Comment:
Why not directly querying the correct tab headers? 
```suggestion
    return this.host.querySelectorAll('& > post-tab-header');
```